src/vivarium_gates_nutrition_optimization_child/data/utilities.py

```diff
@@ -448,7 +453,7 @@ def scrub_neonatal_age_groups(data: pd.DataFrame) -> pd.DataFrame:
     non_neonatal = data.query("age_start >= 1")
     post_neonatal = data.loc[
         (data.index.get_level_values("age_start") >= NEONATAL_END_AGE)
-        & (data.index.get_level_values("age_start") < 1)
+        & (data.index.get_level_values("age_start") < 0.5)
```

Comment:
this seems to have just been meant to get the first post-neonatal age bin, so I changed it from 1 to 0.5